learners/setup.md

```diff
@@ -2,48 +2,32 @@
 title: Setup
 ---
 
-Setup instructions live in this document. Please specify the tools and the data
-sets the Learner needs to have installed.
+## Setup
 
-## Data Sets
+Please perform this setup before you start with the first episode.
 
-Download the [data zip file](data/data.zip) and unzip it to your Desktop
+### MUSCLE3
 
-## Software Setup
+Installing MUSCLE3 on Python will install all the Python-based components of the system, i.e. the Python version of libmuscle, the YMMSL Python library, and the MUSCLE Manager. This requires at least Python 3.6.
 
-::::::::::::::::::::::::::::::::::::::: discussion
+MUSCLE3 is on PyPI as an ordinary Python package, so it can be installed via Pip in the usual way. It’s normally a good idea to make a virtual environment (virtualenv), if you don’t yet have one:
 
-### Details
+```bash
+python3 -m venv muscle3_venv
+. muscle3_venv/bin/activate
+pip install -U pip setuptools wheel
+pip install muscle3
```

Comment:
Without matplotlib the coupled simulation crashes when we try to run it from the `coupled_model.py` file. 
```suggestion
pip install muscle3
pip install matplotlib
```